package.json

```diff
@@ -73,6 +76,7 @@
     "axios-case-converter": "^0.8.1",
     "cidr-tools": "^4.3.0",
     "classnames": "^2.3.1",
+    "esbuild-plugin-copy": "^1.3.0",
```

Comment:
I think this should be a `devDependency`

Reply:
ok!